src/ontology/mondo-edit.obo

```diff
@@ -258204,12 +258221,14 @@ xref: Orphanet:370959 {source="MONDO:subClassOf", source="OMIM:613151"}
 xref: UMLS:C3150412 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo", source="OMIM:613151"}
 is_a: MONDO:0000172 {source="OMIM:613151"} ! muscular dystrophy-dystroglycanopathy, type B
 is_a: MONDO:0018277 {source="ORDO:370959/btnt"} ! congenital muscular dystrophy with cerebellar involvement
+is_a: MONDO:0700068 {source="https://clinicalgenome.org/affiliation/50061/"} ! myopathy caused by mutation in POMGNT1
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/19139 {source="mim2gene_medgen"} ! POMGNT1
 property_value: confidence "1.8985054133659522" xsd:double
 
 [Term]
 id: MONDO:0013156
 name: muscular dystrophy-dystroglycanopathy (congenital without intellectual disability), type B4
+synonym: "congenital muscular dystrophy-FKTN related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/